woocommerce/woocommerce-ios
# Updated tracking for Order Filters

Closes: #6056 

It would be great to merge this PR in our next release 8.5.

### Description
Update tracking for Order filters so that it matches the events and properties defined in the new tracking plan that you can find here p91TBi-6i1-p2.

![Schermata 2022-02-07 alle 11 33 26](https://user-images.githubusercontent.com/495617/152771562-908cd452-4912-4e24-8b16-3ca2b0ff39bb.png)


### Testing instructions
1) Run the app.
2) Go on Orders.
3) Tap the filter button. -> You should see the event `*_order_list_view_filter_options_tapped` in console.
4) Add some filters, and press "Show orders". -> You should see the event `*_orders_list_filter` in console.
5) In the Order list, try to do a search. -> After every search, you should see the event `*_orders_list_search` in console.

---
